scrapy_splash/response.py

```diff
@@ -128,9 +128,6 @@ def data(self):
     def text(self):
         return self._ubody
 
-    def body_as_unicode(self):
-        return self._ubody
```

Comment:
I think it makes sense to keep it, to avoid breaking user code. We can issue a deprecation warning.